Functions/Functions/AuthCMS/NetlifyCallback.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using OidcApiAuthorization.Abstractions;
+using OidcApiAuthorization.Models;
+using System.Configuration;
+using System.Net.Http;
+using System.Text;
+
+namespace SSW.Rules.Functions
+{
+    public class NetlifyCallback
+    {
+
+        [FunctionName("NetlifyCallback")]
+        public async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "callback")] HttpRequest req,
+            ILogger log)
+        {
+            log.LogInformation($"C# HTTP trigger function {nameof(NetlifyCallback)} processed a request.");
+
+            string code = req.Query["code"];
+            string host = req.Headers["host"];
+            log.LogInformation("host: " + host);
+
+            if (string.IsNullOrEmpty(code))
+            {
+                return new BadRequestObjectResult(new
+                {
+                    message = "Missing code param",
+                });
+            }
+
+            log.LogInformation(code);
```

Comment:
Do we want to log the code without any other information or do we want something to explain what we are logging?